src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/UnionOctetsFWTest.java

```diff
@@ -196,11 +198,23 @@ public void shouldSetString1()
         assertAllTestValuesReadCase3(flyweightRO);
     }
 
+    @Test
+    public void shouldSetString1WithStringFW()
+    {
+        int limit = flyweightRW.wrap(buffer, 0, buffer.capacity())
+            .string1(asStringFW("valueOfString1"))
```

Comment:
can you use `new String8FW("valueOfString1")` instead of the custom method here?